### ISSUE TITLE
Minor case about gh-pages. Suggest actual version to download in

### ISSUE THREAD

**Issue by kiddick** [DamnWidget/anaconda#594]
May be it would be nice to have some automatic way to keep download link in actual state here
https://github.com/DamnWidget/anaconda/blob/gh-pages/index.html#L127
![image](https://cloud.githubusercontent.com/assets/9250239/20574669/1def0192-b1cf-11e6-9cb6-445572f2f5bc.png)

**Comment by DamnWidget** [comment]
Yeah, I know, I had something in mind but what I don't have is enough time
